UnityProject/Assets/Scripts/Items/PlaceableTileReplaceBase.cs

```diff
@@ -115,7 +115,8 @@ void ProgressFinishAction()
 				interactableTiles.TileChangeManager.UpdateTile(cellPos, placeableTileEntry.layerTile);
 				interactableTiles.TileChangeManager.SubsystemManager.UpdateAt(cellPos);
 				Inventory.ServerConsume(interaction.HandSlot, placeableTileEntry.itemCost);
-				SoundManager.PlayNetworkedAtPos(placeSound, targetPosition);
+				// JESTER
+				//SoundManager.PlayNetworkedAtPos(placeSound, targetPosition);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)